internal/backend/schedule/schedule.go

```diff
@@ -0,0 +1,98 @@
+package schedule
+
+import (
+	"context"
+	"fmt"
+
+	"go.temporal.io/sdk/client"
+	"go.uber.org/zap"
+
+	"go.autokitteh.dev/autokitteh/internal/backend/dispatcher"
+	"go.autokitteh.dev/autokitteh/internal/backend/temporalclient"
+	"go.autokitteh.dev/autokitteh/internal/kittehs"
+	"go.autokitteh.dev/autokitteh/sdk/sdkservices"
+	"go.autokitteh.dev/autokitteh/sdk/sdktypes"
+)
+
+type temporalScheduleImpl struct {
+	z      *zap.Logger
+	tmprl  temporalclient.Client
+	events sdkservices.Events
+}
+
+type TemporalSchedule interface {
+	CreateScheduledWorkflow(ctx context.Context, scheduleID string, schedule string, triggerID sdktypes.TriggerID) error
+	DeleteSchedulerWorkflow(ctx context.Context, scheduleID string) error
+	UpdateSchedulerWorkflow(ctx context.Context, scheduleID string, schedule string) error
+}
+
+func New(z *zap.Logger, events sdkservices.Events, tc temporalclient.Client) TemporalSchedule {
+	return &temporalScheduleImpl{z: z, events: events, tmprl: tc}
+}
+
+func (tsc *temporalScheduleImpl) CreateScheduledWorkflow(ctx context.Context, scheduleID string, schedule string, triggerID sdktypes.TriggerID) error {
+	event := kittehs.Must1(sdktypes.EventFromProto(&sdktypes.EventPB{EventType: sdktypes.SchedulerEventTriggerType}))
```

Comment:
can you add a NewEvent function that does this?

Reply:
did it, but it's a function of 2 lines, I think it's unnecessary actually